tests/suites/test_suite_oid.function

```diff
@@ -0,0 +1,38 @@
+/* BEGIN_HEADER */
+#include "mbedtls/oid.h"
+#include "mbedtls/asn1.h"
+#include "mbedtls/asn1write.h"
+#include "string.h"
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_OID_C
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE depends_on:MBEDTLS_ASN1_WRITE_C*/
+void oid_get_certificate_policies( char * oid_hex, char * result_str )
+{
+    unsigned char oid[64];
+    mbedtls_asn1_buf asn1_buf= { 0, 0, NULL };
+    int ret;
+    size_t oid_len;
+    const char *desc;
+
+    oid_len = unhexify( oid, oid_hex );
```

Comment:
Could you use a `data_t` instead of `char *` for the oid parameter? Then you don't have to unhexify or stack allocate 64 bytes, etc.